src/spectrecoff-cli/commands/Panel.fs

```diff
@@ -39,4 +39,42 @@ type PanelDocumentation() =
     
     override _.Execute(_context, _) = 
         Cli.Theme.setDocumentationStyle
+
+        NewLine |> toConsole
+        pumped "Panel module"
+        |> alignedRule Left
+        |> toConsole
+
+        Many [
+            CO [
+                C "This module provides functionality from the panel widget of Spectre.Console ("
+                Link "https://spectreconsole.net/widgets/panel"
+                C ")"
+            ]
+            NL
+            C "The panel can be used by the panel function:"
+            BI [P "panel: (header: string) -> (content: OutputPayload) -> Renderable"]
+            NL
+            CO [
+                C "While the content can be an arbitrary payload" 
+                E "(known bug: Many does not work inside panel, fix on the way)"
+                C "the header needs to be a string."
+                C "Spectre still will render markup here, though, so all"; 
+                P "stringifyable payloads"; 
+                C "can be used here when mapped to string using the extension"; 
+                P "payload.toMarkedUpString."
+            ]
+            NL
+            CO [
+                C "This panel will use the"; 
+                P "Panel.defaultPanelLayout,"; 
+                C "initialized to heavy border, collapsed expansipn behaviour and a padding of 2 units on all sides."
+            ]
+            NL
+            C "In order to produce a panel with other layout, the function"
+            BI [P "customPanel: PanelLayout -> string -> OutputPayload -> Renderable"]
```

Comment:
I don't think this is technically correct (same for the normal `panel` above). The function returns `OutputPayload`, with `Renderable` being a union case of it.
You might be aware though, and just wanted to make it clear which case of `OutputPayload` is really returned, in which case I am also fine with it. 

<img width="438" alt="image" src="https://user-images.githubusercontent.com/17160319/205434230-2312e430-1383-44af-b7ce-a90ab15ec754.png">

Reply:
Yeah, I struggled with that one, but you're right. Changed if for the rule documentation as well.